nutella/src/utils/api/Signup.ts

```diff
@@ -30,26 +26,10 @@ export interface InfoType {
 }
 
 export const postUserInfo = async (data: any) => {
-  try {
-    const response = await request.post<InfoType>(
-      Uri.defaultInfomation.get({
-        "Content-Type": "application/json",
-      }),
-
-      data
-    );
-    return response;
-  } catch (error) {
-    console.log(error);
-  }
+  await request.post<InfoType>(Uri.defaultInfomation.get(data));
```

Comment:
```suggestion
  await request.post<InfoType>(Uri.defaultInfomation.get());
```
defaultInfomation은 변수가 없기때문에 get에 매개변수를 넣지 않아도 됩니다.